htmlcss/lesson2/inclass.html

```diff
@@ -41,7 +41,7 @@
 </ol>
 
 <p>My favorite kind of rabbit is a dwarf lop:</p>
-<img src="http://www.pocanticohills.org/4thgrade/06/dwarfloprabbit.jpg">
+<img src="http://www.petplanet.co.uk/petplanet/images/breeds/rabbit_dwarf_lop.jpg">
```

Comment:
Sure, this issue is for the typo. 
The link issue is #298. I will respond at there when I finished the upload.